docs/reference/projectyml.rst

```diff
@@ -54,6 +54,66 @@ The code snippet below provides an example of a complete ``project.yml`` file:
       index_urls:
         - https://pypi.org/simple/
 
+
+.. _specify_location_pip:
+
+
+Specify locations from where V2 modules will be installed
+---------------------------------------------------------
+This section explains how to configure your project in order to download v2 modules from any python package repository.
+By default, the project is configured to install packages from ``https://pypi.org/simple/``. There are multiple ways
+to change this behaviour.
+
+Using the global pip config file
```

Comment:
The global config file is just one type of pip config file, there are multiple of them (https://pip.pypa.io/en/stable/topics/configuration/#configuration-files)
```suggestion
Using pip config file at a standard location
```